packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -720,7 +727,7 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
         {...splitButtonProps}
         data-ktp-execute-target={keytipAttributes ? keytipAttributes['data-ktp-execute-target'] : keytipAttributes}
         onMouseDown={this._onMouseDown}
-        tabIndex={-1}
+        tabIndex={primaryDisabled ? 0 : -1}
```

Comment:
Following the comment above, this should probably be 

```suggestion
        tabIndex={primaryDisabled && !allowDisabledFocus ? 0 : -1}
```